pkg/diff/diff.go

```diff
@@ -687,6 +688,10 @@ func (d *differ) loadLayer(ctx context.Context, node *EventTreeNode, inputIdx in
 		if errors.Is(err, io.EOF) {
 			break
 		}
+		if d.o.CanonicalPaths {
+			hdr.Name = strings.TrimPrefix(hdr.Name, "/")
+			hdr.Name = strings.TrimPrefix(hdr.Name, "./")
```

Comment:
Should this cover paths like `/././.` too ?

Reply:
Technically yes, but that's not common. Maybe can do it later if needed?

Reply:
👍